modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,96 @@
+spec:
+  # (...)
+  config:
+    env:
+      - name: NAMESPACE_SCOPED_ARGO_ROLLOUTS
+        value: 'true' #<1>
+----
+<1> - Specifies the value of the environment variable for enabling namespace-scoped Argo Rollouts installation
+
+. Click *Save*.
++
+As a result, the {gitops-title} Operator enables the namespace-scoped Argo Rollouts installation to reconcile Argo Rollouts CR.
+
+. You can also verify whether the {gitops-title} Operator has enabled namespace-scoped Argo Rollouts installation by viewing the logs of the Openshift GitOps container. 
+.. In the *Administrator* perspective, navigate to *Workloads* → *Pods*. 
+.. Select the `openshift-gitops-operator-controller-manager` Pod, and then select the *Logs* tab.
++
+As a result, the container logs display a log statement "Running in namespaced-scoped mode", which indicates that the {gitops-title} Operator has enabled the namespace-scoped Argo Rollouts installation.
+
+. Create a RolloutManager resource to perform the namespace-scoped Argo Rollout installation. 
+.. Create or select the project where you want to perform the namespace-scoped Argo Rollout installation from the *Project* drop-down menu.
+.. Navigate to *Operators* → *Installed Operators* → *Red Hat OpenShift GitOps*, and select the *RolloutManager* tab.
+.. Click the *Create RolloutManager* button.
+.. Select *YAML view*. When creating a namespace-scoped Argo Rollouts installation, you must set the field '.spec.namespaceScoped:` to `true` in the RolloutManager CR. 
+.. Enter the following snippet in the *YAML view*.
++
+.Example - RolloutManager CR for namespace-scoped Argo Rollouts installation
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: rollout-manager
+spec:
+  namespaceScoped: true
+----
+
+.. Click *Create*. 
++
+After the RolloutManager CR is created, {gitops-title} installs a namespace-scoped Argo Rollouts installation into the namespace.
+
+. To verify the successful installation of namespace-scoped Argo Rollouts installation, in the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the RolloutManager instance shows as *Phase: Available*.
++
+You can also examine the following output in the *YAML* tab under the *RolloutManagers* section to ensure that the installation is successful.
++
+.Example - namespace-scoped Argo Rollouts installation output in the YAML tab
++
+[source,yaml]
+----
+spec:
+  namespaceScoped: true
+status:
+  conditions:
+    lastTransitionTime: '2024-07-10T14:20:5z`
+    message: ''
+    reason: Success
+    status: 'True'
+    type: 'Reconciled'
+  phase: Available
+  rolloutController: Available
+----
++
+This output indicates that namespace-specific Argo Rollouts installation is enabled successfully. For instance, an error message will appear if a namespace-specific Argo Rollouts installation is attempted while a cluster-wide installation already exists on the cluster.
```

Comment:
instead of this, it would be great to add something like the following:
```suggestion
+
You can see an error message if a namespace-specific Argo Rollouts installation is attempted while a cluster-scoped installation already exists on the cluster:
+
.Example of an incorrect installation with an error message
[source,ymal]
----
<Show the example here>
----
```

Reply:
Fixed.